xml/System.Reflection.Metadata/MetadataUpdater.xml

```diff
@@ -64,7 +79,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Returns <see langword="true" /> if the apply assembly update is enabled and available; <see langword="false"/> otherwise.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if the assembly update is enabled and available; otherwise, <see langword="false" />.</value>
```